bus/bus.go

```diff
@@ -322,7 +329,14 @@ func (b *bus) walletPrepareRenewHandler(jc jape.Context) {
 	if jc.Decode(&wprr) != nil {
 		return
```

Comment:
There are a bunch of `Handler writes multiple responses (previous write at D:\a\renterd\renterd\bus\bus.go:322:3)` errors all over the diff.

Reply:
urgh sorry - last minute add